templates/virtualhost.cfg.erb

```diff
@@ -54,3 +54,10 @@ Component "<%= name %>" <% if component.include?('type') then %>"<%= component['
   <%- end -%>
 <% end -%>
 <% end -%>
+<%- if @disco_items != nil -%>
```

Comment:
not tested, but something like this?

```suggestion
<%- unless @disco_items.empty? -%>
```

Reply:
solved in 91da1e311bb6e945937bca205d38e033635f574e